src/System.Windows.Forms.Primitives/tests/UnitTests/Interop/PARAMTests.cs

```
@@ -32,8 +32,11 @@ public unsafe void FromLowHighUnsigned_x32_Result()
         [ConditionalFact(typeof(ArchitectureDetection), nameof(ArchitectureDetection.Is64bit))]
         public unsafe void FromLowHighUnsigned_x64_Result()
         {
-            Assert.Equal((long)0x0000000003040102, (long)PARAM.FromLowHighUnsigned(0x0102, 0x0304));
-            Assert.Equal((long)0x00000000F3F4F1F2, (long)PARAM.FromLowHighUnsigned(0xF1F2, 0xF3F4));
+            if (Environment.Is64BitProcess)
```

Just silencing a failure in TestExplorer- it doesn't know to filter these conditions.